src/Files.App/Keyboard/Actions/ToggleLayoutColumnsAction.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Files.App.Extensions;
+using Files.App.ViewModels;
+using Windows.System;
+
+namespace Files.App.Keyboard.Actions
+{
+	internal class ToggleLayoutColumnsAction : IKeyboardAction
```

Comment:
We should probably group the layout commands together but doesn't have to be right away as it's easy to make this change.

Reply:
What do you mean ? The parameters should allow you to find an order quickly (filter textbox, categories, ...). Is this?

Reply:
Meaning the class should be `ToggleLayoutCommands` and there should be separate methods for `Columns` and `Details` etc.

Reply:
This goes against the interface and forces the manager to make special cases for the various orders. What's wrong with having one class per action? If we add a layout, it will suffice to add an action, without the manager knowing anything about it. There will be hundreds of actions, all with the same minimal interface.

Reply:
There isn't anything wrong but it's also a good idea to group related tasks, at the very least, they should have their own folder.

Reply:
Yes I will add folders.